### DISCUSSION TITLE
Elaborate on the need for using an explicit ID when using multiple flows in documentation

### DISCUSSION THREAD

**Discussion by plc-dev** [bcakmakoglu/vue-flow#1500] [Ideas]
Hi, first of, thanks for the great port of the React flow-library!
I'm not sure if a feature request is the appropriate issue type. Beg my pardon if this is not the case.

When implementing multiple flows for my application, I had to explicitly assign separate IDs for the flow-instances and when accessing the internal state through `useFlow`.
While this makes absolute sense, it took me a bit to figure out the correct setting, as some bugs* occur when not providing separate IDs manually, which led me down the wrong debugging path initially.

Maybe this is worth its own additional example in the documentation of the MultiFlow-example? I'd be willing to provide the additional documentation if you think this would be helpful.

\* e.g. "overriding" the mouse-controls and state of the other instance, etc.

**Comment by bcakmakoglu** [comment]
You don't necessarily need to assign an id, it's a matter of where you call `useVueFlow` the first time (subject to change in the next major to avoid this sort of confusion).
[Read this section of the docs for more about this.](https://vueflow.dev/guide/composables.html#state-creation-and-injection).
